.teamcity/_self/builds/WPComPlugins_EditorToolkit.kt

```diff
@@ -114,27 +102,59 @@ object WPComPlugins_EditorToolKit : BuildType({
 				yarn lint:php
 			"""
 		}
+		/**
```

Comment:
The comment provides a super clear explanation, thanks for keeping inline documentation in mind, I appreciate that.